examples/event_handler_rest/src/split_route_prefix_module.py

```diff
@@ -13,7 +13,7 @@
 @router.get("/")
 @tracer.capture_method
 def get_todos():
-    api_key: str = router.current_event.get_header_value(name="X-Api-Key", case_sensitive=True, default_value="")
+    api_key = router.current_event.headers["X-Api-Key"]
```

Comment:
How about using `.get()`? So we prevent the customer from having problems with `KeyError`.